src/dftbp/api/mm/apicallbackc.f90

```diff
@@ -0,0 +1,80 @@
+!--------------------------------------------------------------------------------------------------!
+!  DFTB+: general package for performing fast atomistic simulations                                !
+!  Copyright (C) 2006 - 2022  DFTB+ developers group                                               !
+!                                                                                                  !
+!  See the LICENSE file for terms of usage and distribution.                                       !
+!--------------------------------------------------------------------------------------------------!
+
+!> Contains types for the callback interface that exports density matrix, overlap, and hamiltonians
+module dftbp_apicallbackc
```

Comment:
The class is pretty general. It can  be extended to export/import something else. For example wavefunctions. Then it will be necessary to rename it again. There is no other API in DFTB+ (isn't it?), so `apicallback` is as specific, as can be.

Reply:
OK, I see, so you think about it as a general callback container, containing all callbacks possible in the code? That is probably a very neat idea, indeed. So, leave the name as it is for the moment. (At some point, in a future PR, we would have to merge it then with the SCC-charges callback.)